src/redisearch_rs/buffer/src/lib.rs

```diff
@@ -121,3 +121,16 @@ impl Buffer {
         self.0.offset += n;
     }
 }
+
+impl std::fmt::Debug for Buffer {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut debug = f.debug_struct("Buffer");
+        let debug = debug
+            .field("len", &self.len())
+            .field("capacity", &self.capacity());
+        // We don't want to accidentally output huge or sensitive data in production code.
+        #[cfg(test)]
```

Comment:
Would `debug_assertions` be a better gate?

Reply:
Probably as this could be helpful when debugging the full module directly.